todoapp/app/src/main/java/com/example/android/architecture/blueprints/todoapp/taskdetail/TaskDetailFragment.kt

```diff
@@ -32,25 +32,27 @@ import com.example.android.architecture.blueprints.todoapp.util.setupSnackbar
 /**
  * Main UI for the task detail screen.
  */
-class TaskDetailFragment : LifecycleFragment() {
+class TaskDetailFragment : Fragment() {
 
     private lateinit var viewDataBinding: TaskdetailFragBinding
 
     override fun onActivityCreated(savedInstanceState: Bundle?) {
         super.onActivityCreated(savedInstanceState)
         setupFab()
-        view?.setupSnackbar(this, viewDataBinding.viewmodel.snackbarMessage, Snackbar.LENGTH_LONG)
+        viewDataBinding.viewmodel?.let {
+            view?.setupSnackbar(this, it.snackbarMessage, Snackbar.LENGTH_LONG)
+        }
     }
 
     private fun setupFab() {
         with(activity.findViewById<View>(R.id.fab_edit_task)) {
```

Comment:
`with` is not necessary here for the setup.

Reply:
Done.